scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_1.txt

```diff
@@ -0,0 +1,33 @@
+Frequently Asked Questions (FAQ)
+New users should consult the Introduction to HPC
+to get started, which is a great resource for learning the basics, troubleshooting, and looking up specifics.
+If you want to use software that's not yet installed on the HPC, send us a
+software installation request.
+Overview of HPC-UGent Tier-2 infrastructure
+Composing a job
+How many cores/nodes should I request?
+An important factor in this question is how well your task is being parallelized:
+does it actually run faster with more resources? You can test this yourself:
+start with 4 cores, then 8, then 16... The execution time should each time be reduced to
+around half of what it was before. You can also try this with full nodes: 1 node, 2 nodes.
+A rule of thumb is that you're around the limit when you double the resources but the
+execution time is still ~60-70% of what it was before. That's a signal to stop increasing the core count.
+See also: Running batch jobs.
+Which packages are available?
```

Comment:
maybe add in special comment for the LLM soem alias questions, like `What software is avaialbel`